### ISSUE TITLE
Generate `index.html` from `static-index.jsx`

### ISSUE THREAD

**Issue by coopy** [FormidableLabs/victory-component-boilerplate#20]
Use [`renderToString`](https://facebook.github.io/react/docs/top-level-api.html#reactdomserver.rendertostring) to generate `index.html` from `static-index.jsx`. This is to support `builder run docs-dev` and `builder run docs-hot` scenarios.

**Comment by coopy** [comment]
Depends on https://github.com/FormidableLabs/builder/issues/68

**Comment by ryan-roemer** [comment]
Suggest the easier version of:

``` js
"docs-dev": "builder run build-index && builder run real-docs-dev"
```

OR

``` js
"docs-dev": "builder --setup=build-index run real-docs-dev"
```

**Comment by coopy** [comment]
Yeah, let's just support this by modifying the `docs-dev` and `docs-run` scripts in the archetype.
